layouts/css/page-modules/_header.styl

```diff
@@ -24,7 +24,7 @@ header
     a:hover
       background-color transparent
       text-decoration underline
-
+  
```

Comment:
Unintentional white-space change?

Reply:
(Just asking about the white space. If someone wants to land this anyway, go for it.)

Reply:
Shouldn't be there, let me rebase.